### PR TITLE
[300] add unit test to mapper from 25 to 36

### PR DESCRIPTION
**Pull request by DmytroLysenko1**

### CODE REVIEW

**Review by vulook — APPROVED** [review]

File: service/src/test/java/greencity/mapping/LanguageTranslationDtoMapperTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+        String expectedLanguageCode = expected.getLanguage().getCode();
+        String actualLanguageCode = actual.getLanguage().getCode();
+
+        Long expectedLanguageId = expected.getLanguage().getId();
+        Long actualLanguageId = actual.getLanguage().getId();
+
+        // Asser
+        assertEquals(expectedLanguageCode, actualLanguageCode);
+        assertEquals(expectedLanguageId, actualLanguageId);
```
Comment (VitaliiChernenko) [inline service/src/test/java/greencity/mapping/LanguageTranslationDtoMapperTest.java:37]: When I tried to do assertEquals(actual, expected); the test failed.
Please review the test.

File: service/src/test/java/greencity/mapping/ShoppingListItemDtoMapperTest.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        String expectedText = expected.getText();
+        String actualText = actual.getText();
+
+        String expectedStatus = expected.getStatus();
+        String actualStatus = actual.getStatus();
+
+        // Assert
+        assertEquals(expectedId, actualId);
+        assertEquals(expectedStatus, actualStatus);
+        assertEquals(expectedText, actualText);
```
Comment (VitaliiChernenko) [inline service/src/test/java/greencity/mapping/ShoppingListItemDtoMapperTest.java:46]: You can use assertEquals(actual, expected); instead of all those asserts. It will be more readable.

File: service/src/test/java/greencity/mapping/ShoppingListItemResponseDtoMapperTest.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+        Long expectedId = expected.getId();
+        Long actualId = actual.getId();
+
+        List<ShoppingListItemTranslationDTO> expectedTranslation = expected.getTranslations();
+        List<ShoppingListItemTranslationDTO> actualTranslation = actual.getTranslations();
+
+        // Assert
+        assertEquals(expectedId, actualId);
+        assertEquals(expectedTranslation, actualTranslation);
```
Comment (VitaliiChernenko) [inline service/src/test/java/greencity/mapping/ShoppingListItemResponseDtoMapperTest.java:51]: Also can be used assertEquals(actual, expected);

File: service/src/test/java/greencity/mapping/ShoppingListItemRequestDtoMapperTest.java [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+                .build();
+
+        // Act
+        UserShoppingListItem actual = mapper.convert(shoppingListItemRequestDto);
+
+        Long expectedId = expected.getId();
+        Long actualId = actual.getId();
+
+        // Assert
+        assertEquals(expectedId, actualId);
```
Comment (VitaliiChernenko) [inline service/src/test/java/greencity/mapping/ShoppingListItemRequestDtoMapperTest.java:40]: Why are you comparing ids?
It will be better to compare objects. Objects are the result of mapper work.
You can use assertEquals(expected, actual); as You used in all your tests before. Why did you change your style?

File: service/src/test/java/greencity/mapping/ShoppingListItemWithStatusRequestDtoMapperTest.java [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+        ShoppingListItemStatus expectedStatus = expected.getStatus();
+        ShoppingListItemStatus actualStatus = actual.getStatus();
+
+        ShoppingListItem expectedShoppingListItem = expected.getShoppingListItem();
+        ShoppingListItem actualShoppingListItem = actual.getShoppingListItem();
+
+        // Assert
+        assertEquals(expectedStatus, actualStatus);
+        assertEquals(expectedShoppingListItem, actualShoppingListItem);
```
Comment (VitaliiChernenko) [inline service/src/test/java/greencity/mapping/ShoppingListItemWithStatusRequestDtoMapperTest.java:42]: The same issue. You sould compare assertEquals(expected, actual); because the actual in this case is a result of mapper work. And we need to check if it works completely right not only some fields.

File: service/src/test/java/greencity/mapping/TagDtoMapperTest.java [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+        Long expectedId = expected.getId();
+        Long actualId = actual.getId();
+
+        String expectedName = expected.getName();
+        String actualType = actual.getName();
+
+        // Assert
+        assertEquals(expectedId, actualId);
+        assertEquals(expectedName, actualType);
```
Comment (VitaliiChernenko) [inline service/src/test/java/greencity/mapping/TagDtoMapperTest.java:48]: Please check assertEquals(actual, expected); instead of those fields.

**Review by ZenukD — APPROVED** [review]

**Review by DmitryKhomich — APPROVED** [review]

**Review by NazarVavrushchak — APPROVED** [review]
